docs/public/components/overview.md

```diff
@@ -5,68 +5,35 @@ is invoked, different components may be instantiated.
 
 ## What is a Component?
 
-Any well-defined portion of the codebase, with a clearly documented API, _should_ be a component. The ultimate goal of
-any component is to clearly encapsulate the logic/feature block that composes the Datadog Agent. Components can be small
-or very large but all share the same goals.
+A component encapsulates a particular piece of logic/feature in a clear and documented interface. 
 
 A component must:
 
-* **Limit blast radius**:
-    + Have a single interface clearly highlighting how the component must be used and what features it offers.
-    + Encapsulate and abstract all the complexity of its subject. As much as possible a user of a component should not
-      have to know all the internals of it. As well, any change to the internals of a component should have no
-      impact on the code using it.
-    + Clearly list its dependencies, which could be passed using dependency injection (more on this below).
-* **Increase development velocity**:
-    + All components must be reusable no matter the context or the binary in which they're included.
-    + Expose a mock implementing their interface to ease testing.
-    + As long as its dependencies are present a component should work the same no matter in which context it's used.
-      This means we should be able to easily create new binary/agents by composing different components.
-    + The interface must be carefully documented allowing developers from other team to easily reuse it.
-    + Any change within a component, that don't change its interface, should not require QA of the other component using
-      it.
-* **Clear ownership and responsibility**:
-    + Each component should be owned by a single team which will support and maintain it.
+  + The component hides the complexity of the implementation from its users.
+  + All components must be reusable no matter the context or the binary in which they're included.
+  + A component must be tested.
+  + Expose a mock implementation to help testing.
+  + Any change within a component, that don't change its interface, should not require QA of the other component using
+    it.
+  + Each component should be owned by a single team which will support and maintain it.
```

Comment:
Providing a more condense list is best to capture the reader attention. In the creating a component page we can mention any other point that is missing from the overview page 😄

Reply:
I felt that the 3 sections highlighted clearly why we're doing the whole component thing. Explaining what must be done it good, but we're trying to convince people, so the why is also important.